app/models/policy_manager/concerns/user_behavior.rb

```diff
@@ -130,4 +130,12 @@ def can_request_portability?
     self.portability_requests.select{|p| p.pending? || p.progress?}.blank?
   end
 
+  def store_cookie(name)
```

Comment:
@patriciojofre  why call it `store_cookie` if it can apply to any rule ?
We could have a `store_policy_RULE_NAME` for every rule initialized in the user concern for example
What do you think ?

Reply:
I agree with the method naming, however I think we could avoid creating multiple methods and just just add the `store_policy_from(name)`. 
This is just a method that wraps the "current_user.handle_policy_for(@term)" with an extra logic were validates the presence of a published term.

Reply:
since the method accepts a term, we should name it as `accept_policy_from`

Reply:
+1 @michelson, create multiples methods for this little wrapper is not necessary.
I'll change the name to `accept_policy_from` 😄